common-before.js

```diff
@@ -62,26 +63,23 @@ const PHASE_ICON_MAP = new Map([
     [DRAW_PHASE, "Draw Phase Icon"],
     [END_PHASE, "End Phase Icon"],
 ]);
-// Labels for indented effects (power, reaction) and (less gross) regex
+// Labels for indented effects (power, reaction, bullets):
 const POWER_LABEL = "power:";
 const REACTION_LABEL = "reaction:";
-const INDENT_LABELS = [POWER_LABEL, REACTION_LABEL];
-const INDENT_REGEX = new RegExp(`^(${INDENT_LABELS.join("|")})`);
+const BULLET_LABEL = "»";
+const INDENT_LABEL_MAP = new Map([
```

Comment:
Idle thought, wonder if there's some tool out there that can convert all the maps in the code to a table for documentation. It'd be nice to be able to show people: "this is what the text you typed in turned into and why".